todo_app/app.py

```diff
@@ -1,22 +1,24 @@
 from flask import Flask,  render_template
 from todo_app.data import session_items
 from todo_app.flask_config import Config
-
-list = ['Harry Potter', 'Sherlock Holmes', 'Dino Master']
+from flask import request
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
-
 @app.route('/')
 def index():
-    return render_template('index.html', list = list)
+    list = session_items.get_items()
+    return  render_template('index.html', list=list)
 
 @app.route('/addTitles')
 def addTitle():
     return session_items.add_item('Never Land')
-def getTitle():
-    return session_items.get_item('4')         
+    
+@app.route('/add', methods=['POST'])
+def add():
+	item = request.form["title"]
+	item.add()
+	return redirect('/')        
 
-if __name__ == '__main__':
-    app.run()
+app.run()
```

Comment:
Either delete this line, or put it back inside `if __name__ == '__main__':` 

When starting the app via `poetry run flask run`, it will detect the app variable and run it - no need to call app.run()